packages/public/inspection-capture-web/src/PhotoCaptureHUD/PhotoCaptureHUDPreview/PhotoCaptureHUDPreview.tsx

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react';
+import { useObjectTranslation } from '@monkvision/common';
+import { useTranslation } from 'react-i18next';
+import { LabelDictionary } from '@monkvision/types';
+import { Button } from '@monkvision/common-ui-web';
+import { PhotoCaptureHUDPreview, usePhotoCaptureHUDPreview } from './hook';
+import { PhotoCaptureHUDSightsSlider } from './PhotoCaptureHUDSightsSlider';
+import { PhotoCaptureHUDCounter } from './PhotoCaptureHUDSightsCounter';
+
+export interface PhotoCaptureHUDPreviewProps {
+  sights: LabelDictionary;
```

Comment:
The `sights` prop should actually be a list of sights to display at the bottom. The type of this prop should be `Sight[]`